timemachine/fe/mle.py

```diff
@@ -70,7 +70,7 @@ def wrapped(x):
     return wrapped
 
 
-def infer_node_vals(edge_idxs, edge_diffs, edge_stddevs, ref_node_idxs, ref_node_vals):
+def infer_node_vals(edge_idxs, edge_diffs, edge_stddevs, ref_node_idxs=[], ref_node_vals=[]):
```

Comment:
I would avoid mutable data structures as defaults. Empty tuples would be safer

Reply:
Applied in https://github.com/proteneer/timemachine/pull/941/commits/71a963a69329836bdd871b791cf6202b5b97e0f2